source/Windows.Devices.I2c/I2cController.cs

```diff
@@ -14,26 +15,24 @@ namespace Windows.Devices.I2c
 	public sealed class I2cController
     {
         // this is used as the lock object 
-        // a lock is required because multiple threads can access the I2C controller
-        readonly static object _syncLock = new object();
+        // a lock is required because multiple threads can access the I2cController
+        [System.Diagnostics.DebuggerBrowsable(System.Diagnostics.DebuggerBrowsableState.Never)]
+        private static readonly object _syncLock = new object();
```

Comment:
Must the lock object be static so that multiple I2cController objects would lock one another eventually? Or would be one lock object per controller be better (if only this controller needs a lock and not the whole I²C system)?

Reply:
```suggestion
        private readonly object _syncLock = new object();
```

Reply:
It doesn't have to be static. Removed that.